### PR TITLE
fix(schematics): skip e2e tsconfig for add to non e2e project

### LINKED ISSUE #453: Not existing `e2e/tsconfig.json` referenced for new Angular 12 app
When adding `angular-eslint` to a newly created Angular 12 app the `parserOptions` contains a reference to `e2e/tsconfig.json`, which does not exist since the Angular team removed a default e2e setup.

```
"parserOptions": {
        "project": [
          "tsconfig.json",
          "e2e/tsconfig.json"
        ],
```

Expected: Don't add the file if no e2e setup exists?

### PR DESCRIPTION
**Pull request by GavinWu1991**
fix #453 
The pr contains the change to determine if the project contains e2e content by checking for the presence of e2e architect in the `angular.json`, then skip adding e2e's tsconfig to`parserOptions.projects` if the target project is a non-e2e project.

### CODE REVIEW

File: packages/schematics/src/utils.ts [hunk 1]
```diff
@@ -396,3 +418,15 @@ export function determineTargetProjectName(
...
+
+/**
+ * Checking if the target project has e2e setup
+ * Method will check if angular project architect has e2e configuration to determine if e2e setup
+ */
+export function determineTargetProjectHasE2E(
+  angularJSON: any,
+  projectName: string,
+): boolean {
+  const { architect } = angularJSON.projects[projectName];
```
Comment (JamesHenry) [inline packages/schematics/src/utils.ts:430]: `architect` can also be called `targets`, we already have a utility function to normalize this within this file called `getTargetsConfigFromProject()` so you just need to use that here

File: packages/schematics/src/utils.ts [hunk 2]
```diff
@@ -396,3 +418,15 @@ export function determineTargetProjectName(
...
+  return Object.prototype.hasOwnProperty.call(architect, 'e2e');
```
Comment (JamesHenry) [inline packages/schematics/src/utils.ts:431]: Nit: we're dealing with a very simple object here - with its origins in `.json` - and no prototypal chain to consider, so a simple `!!architect.e2e` or `Boolean(architect.e2e)` would have also been fine

**Review by JamesHenry — CHANGES_REQUESTED** [review]
Thanks! Just one point to change and one nit

File: packages/schematics/src/utils.ts (hunk 2 above)
Comment (GavinWu1991) [reply packages/schematics/src/utils.ts]: Got it, I simplified the determination logic. Thx

**Comment by JamesHenry** [comment]
Thanks @GavinWu1991!

**Pull request merged**